src/fr/invinciblecomics/src/eu/kanade/tachiyomi/extension/fr/invinciblecomics/InvincibleComics.kt

```diff
@@ -0,0 +1,132 @@
+package eu.kanade.tachiyomi.extension.fr.invinciblecomics
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Element
+
+class InvincibleComics : HttpSource() {
```

Comment:
Use the <code>ParsedHttpSource</code> instead of <code>HttpSource</code> 
```suggestion
class InvincibleComics : ParsedHttpSource() {
```

Reply:
We are gradually deprecating `ParsedHttpSource`. Using `HttpSource` is encouraged.